### PR TITLE
update olive

### PR DESCRIPTION
**Pull request by Friedrich2001**
This is my first contibution to flathub, so please tell me if i made a mistake somewhere.

expat and imath Modules where copied from [Krita Flatpak](https://github.com/flathub/org.kde.krita)
portaudio Module was copied from [Audacity Flatpak](https://github.com/flathub/org.audacityteam.Audacity)

### CODE REVIEW

**Comment by hfiguiere** [comment]
This a change to the beta branch that was obviously way behind but you never resync the changes from master/stable, thus you even build was a runtime that is EOL.

**Comment by hfiguiere** [comment]
Still you didn't merge the changes from the master branch to beta. Notably you still disable the aarch64 build.

**Comment by hfiguiere** [comment]
bot, build

**Comment by hfiguiere** [comment]
I had a few patches for these GL error on aarch64, that can be found on the master branch. Don't know if they apply on this.

File: org.olivevideoeditor.Olive.yaml [hunk 1]
```diff
@@ -161,19 +196,44 @@ modules:
...
     buildsystem: cmake-ninja
     builddir: true
     config-opts:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DUSE_WERROR=OFF
     sources:
       - type: git
         url: https://github.com/olive-editor/olive.git
-        commit: 0699795c6f2e81d7c1594f9815dde11468e9c791
+        commit: 55d3915881b36161db6240d7e3ad7decc4730337
```
Comment (hfiguiere) [inline org.olivevideoeditor.Olive.yaml:237]: this commit doesn't seem to match any release, and you don't provide any info

Comment (Friedrich2001) [reply org.olivevideoeditor.Olive.yaml:237]: Olive 0.2 has new nightly builds after every commit. https://www.olivevideoeditor.org/nightly.php

Comment (hfiguiere) [reply org.olivevideoeditor.Olive.yaml:237]: but flathub isn't really for "nightlies"

Comment (Friedrich2001) [reply org.olivevideoeditor.Olive.yaml:237]: I might be wrong but i think commit 0699795c... is also just a nightly build.

**Pull request merged**